tests/jerry/es.next/proxy-evil-recursion.js

```diff
@@ -0,0 +1,166 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Copyright 2015 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
I think that this should be handled. Copy-pasting these 3 lines here has no real meaning as there is no LICENSE file that is referred. Most probably the whole contents of the original LICENSE file should be copied here, similarly to jerry-core/ecma/base/ecma-helpers-errol.c or jerry-math/acos.c .

Reply:
Ah sorry about this. The test was not copied from v8. I've created it. I just copied the comment block from another test file :(.

Reply:
Well and wow. We may need to check that test file (those test files??) then.

Reply:
I can agree that we can visit these tests, but in a separate PR.